partials/featured-by.hbs

```diff
@@ -1,4 +1,4 @@
-<section class="featuredby-wrapper container">
+<section class="featuredby-wrapper container pt-2">
```

Comment:
Could this be increased to around `pt-8`? Currently I think the padding increase isn't really visible.

For ref., `pt-2` [means top padding of 8px](https://tailwindcss.com/docs/padding), which looks like this:

![image](https://github.com/AdvisorySG/dawn-advisory-theme/assets/2070423/7bdae8c8-cd33-41cf-8c6b-617268293824)